Tests/SwiftSyntaxBuilderTest/ExpressibleBuildablesTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+import SwiftSyntax
+
+import SwiftSyntaxBuilder
+
+final class ExpressibleBuildablesTests: XCTestCase {
+    func testExpressibleAsMemberDeclListItem() {
+        let myStruct = StructDecl(identifier: "MyStruct", members: MemberDeclBlock(membersBuilder: {
+            VariableDecl(.var, name: "myFirstVar", type: "Int")
```

Comment:
This is really starting to look like you can generate real-world code with `SwiftSyntaxBuilder` 😍

Reply:
Yes! It look really good and it gets more and more clean to read! 🎉